snakemake/benchmark.py

```diff
@@ -237,7 +237,10 @@ def _update_record(self):
                             this_time - self.bench_record.prev_time
                         )
 
-                    meminfo = proc.memory_full_info()
+                    try:
+                        meminfo = proc.memory_full_info()
+                    except psutil.Error:
```

Comment:
May I ask you to add your explanation from the PR text as a rationale here? So that we know in the future why we catch and continue here.

Reply:
done

Reply:
Also, any idea about the reason for this starter-suid thing showing up? proc should not point to a surrounding process... Why does that happen?

Reply:
Yes, setting to NA seems like the most transparent way to me, and also display a warning via `logger.warning()`.

Reply:
New commit that should handle this have been pushed.